AD LDS & ADAM do not have have `userAccountControl` attribute present. (#12721)
github/docs: content/admin/authentication/authenticating-users-for-your-github-enterprise-server-instance/using-ldap.md

{% note %}

> **Note**: LDAP entries can only be marked as disabled if you use Active Directory and the `userAccountControl` attribute is present and flagged with `ACCOUNTDISABLE`. Some variations of Active Directory, such as AD LDS and ADAM, do not have the `userAccountControl` attribute present.

Reviewer: Thanks for this contribution @affrae !

I'll make one tiny change to the language and :shipit: 

```suggestion
**Note**: LDAP entries can only be marked as disabled if you use Active Directory and the `userAccountControl` attribute is present and flagged with `ACCOUNTDISABLE`. Some variations of Active Directory, such as AD LDS and ADAM, don't support the `userAccountControl` attribute.
```

Author: Sounds great - thank you!
